common/src/main/java/org/conscrypt/OpenSSLCipherRSA.java

```diff
@@ -441,7 +441,9 @@ protected AlgorithmParameters engineGetParameters() {
                                 EvpMdRef.getJcaDigestAlgorithmStandardNameFromEVP_MD(mgf1Md)),
                         pSrc));
                 return params;
-            } catch (NoSuchAlgorithmException | InvalidParameterSpecException e) {
+            } catch (NoSuchAlgorithmException e) {
+                throw new RuntimeException("No providers of AlgorithmParameters.OAEP available");
```

Comment:
Yeah, it's a checked exception so we have to grab it.  AssertionError() is maybe more appropriate here, since it should be impossible.